pkg/frontend/admin_platformworkloadidentityroleset_list.go

```diff
@@ -0,0 +1,49 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"net/http"
+	"path/filepath"
+	"sort"
+
+	"github.com/coreos/go-semver/semver"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/api/admin"
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+)
+
+func (f *frontend) getAdminPlatformWorkloadIdentityRoleSets(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	log := ctx.Value(middleware.ContextKeyLog).(*logrus.Entry)
+	r.URL.Path = filepath.Dir(r.URL.Path)
+
+	converter := f.apis[admin.APIVersion].PlatformWorkloadIdentityRoleSetConverter
+
+	docs, err := f.dbPlatformWorkloadIdentityRoleSets.ListAll(ctx)
+	if err != nil {
```

Comment:
this error is getting swallowed - can we at least log it if we aren't returning it directly? Is there a reason to _not_ return the error in the HTTP response on `/admin`?

Reply:
To be frank, I copied and pasted `pkg/frontend/admin_openshiftversion_list.go` and tweaked it, and this is a less-than-ideal piece that got copied over that I didn't pay much attention to.

+1 to your comment though, and I'll fix it in this file, but I'm not going to fix it in the other file to avoid scope creep. We can add a Jira to the backlog for making similar logging improvements to other endpoints.